### PR TITLE
libwnck3: 3.30.0 → 3.32.0

### PR DESCRIPTION
**Pull request by jtojnar**
https://gitlab.gnome.org/GNOME/libwnck/blob/3.32.0/NEWS#L1-33

### CODE REVIEW

File: pkgs/development/libraries/libwnck/3.x.nix [hunk 1]
```diff
@@ -1,31 +1,67 @@
...
+    docbook_xsl
+    docbook_xml_dtd_412
+  ];
 
-  nativeBuildInputs = [ pkgconfig intltool gobject-introspection ];
-  propagatedBuildInputs = [ libX11 gtk3 ];
+  propagatedBuildInputs = [
+    libX11
+    gtk3
+    libstartup_notification
```
Comment (worldofpeace) [inline pkgs/development/libraries/libwnck/3.x.nix]: Isn't this `Requires.private`? Isn't `glib` in `Requires` too.

Comment (jtojnar) [reply pkgs/development/libraries/libwnck/3.x.nix]: I knew I was forgetting something. Thanks.

File: pkgs/development/libraries/libwnck/3.x.nix [hunk 2]
```diff
@@ -1,31 +1,72 @@
...
+    pkgconfig
+    gettext
+    gobject-introspection
+    gtk-doc
+    docbook_xsl
+    docbook_xml_dtd_412
+  ];
 
-  configureFlags = [ "--enable-introspection" ];
+  buildInputs = [
```
Comment (worldofpeace) [inline pkgs/development/libraries/libwnck/3.x.nix]: oop, really quick.

I see `pango cairo` in `Requires.private` so it probably should be here formally

Comment (jtojnar) [reply pkgs/development/libraries/libwnck/3.x.nix]: Done, thanks.

**Review by worldofpeace — APPROVED** [review]
Built all dependents.

**Pull request merged**